applications/nrf5340_audio/broadcast_sink/Kconfig.defaults

```diff
@@ -61,6 +61,19 @@ config NVS_LOG_LEVEL
 config BT_DEVICE_NAME
 	default "NRF5340_BIS_HEADSET"
 
+## PACS related configs ##
+config BT_PAC_SNK_NOTIFIABLE
+	default y
+
+config BT_PAC_SNK
+	default y
+
+config BT_PAC_SRC_NOTIFIABLE
+	default y
+
+config BT_PAC_SRC
+	default y
```

Comment:
More default overwrites :(

Reply:
This is a temporary fix for S platform, those are optional in the specification but S platform check it. And also in later we will use specific config instead of Kconfig.defaults.